src/flb_conditionals.c

```diff
@@ -66,8 +66,14 @@ static struct flb_condition_rule *rule_create(const char *field,
     switch (op) {
     case FLB_RULE_OP_EQ:
     case FLB_RULE_OP_NEQ:
+        /* Allow empty string values for equality comparisons */
+        if (!value) {
```

Comment:
Please modify this to explicitly compare with `NULL`.

Reply:
Also, wouldn't this be covered by the conditional in line 62? (which should be modified to explicitly compare the pointer values with `NULL`)